src/alert/src/Alert.js

```diff
@@ -21,24 +33,20 @@ const Alert = memo(
       ...restProps
     } = props
 
-    const {
-      tokens: { intents }
-    } = useTheme()
-
     const intentToken = intent === 'none' ? 'info' : intent
-    const className = useAlertApperance(intentToken)
+    const { className: themedClassName, ...styleProps } = useStyleConfig(
+      'Alert',
+      { ...props, appearance, intent: intentToken },
```

Comment:
Would like to pass `props` as-is but unfortunately we have different intents (`none` and `info`)... Thoughts on allowing for both? It's a simple update in the theme to add tokens for `none` and `info` that are the same.